lib/widgets/autocomplete.dart

```diff
@@ -311,16 +311,26 @@ class _EmojiAutocompleteItem extends StatelessWidget {
       ? candidate.emojiName
       : [candidate.emojiName, ...candidate.aliases].join(", "); // TODO(#1080)
 
+    // TODO(design): emoji autocomplete results
+    // There's no design in Figma for emoji autocomplete results.
+    // Instead we adapt the design for the emoji picker to the
+    // context of autocomplete results as exemplified by _MentionAutocompleteItem.
+    // That means: emoji size, text size, text line-height from emoji picker;
+    // text color (for contrast with background) and outer padding
+    // from _MentionAutocompleteItem; padding around emoji glyph
+    // to bring it to same size as avatar in _MentionAutocompleteItem.
```

Comment:
The text line-height in the Figma emoji picker is 24, not 18, so the code or the comment should change:

<img width="806" alt="image" src="https://github.com/user-attachments/assets/371b3a43-574a-407f-aa51-ff2d628c6b7a">

Also worth mentioning that the font weight comes from the emoji picker too (normal weight).

Reply:
> The text line-height in the Figma emoji picker

Hmm, good catch! It's 18 in this other box:
![image](https://github.com/user-attachments/assets/99c962ae-75ca-476c-853d-51e3c097e74b)

That one has the text wrapping onto two lines. It's the same 18px in the other example box where that happens, and 24px in those that fit on one line.

I don't think it's practical to have the line-height vary based on whether the text wraps. That's something the text shaping engine decides, and so isn't yet known at the build stage for a given frame, which is when we decide the line-height.

The line-height mostly ends up mattering when there's more than one line, I think — when there's just one line, it controls the height of the Text widget, but the emoji is taller so that doesn't much affect the layout. (It might slighly affect the vertical alignment as the text is centered.) So let's go with the 18px that's specified for the two-line case.

Reply:
Also started a chat thread to let Vlad know: https://chat.zulip.org/#narrow/channel/48-mobile/topic/variable.20line-height/near/1994856

Reply:
Sure, sounds good.

Reply:
Ah did you see this part:

> Also worth mentioning that the font weight comes from the emoji picker too (normal weight).